src/components/entity/ha-entity-attribute-picker.ts

```diff
@@ -80,6 +80,7 @@ class HaEntityAttributePicker extends LitElement {
         .value=${this._value}
         .allowCustomValue=${this.allowCustomValue}
         .renderer=${rowRenderer}
+        attr-for-value="bind-value"
```

Comment:
Without that line, the value was not shown in the input until you clicked into the field. All existing usages of `<ha-entity-attribute-picker>` in the code base set the attribute `allow-custom-value`. With that set, it works fine, but without this additional option was required.